pytest_slack/plugin.py

```diff
@@ -6,6 +6,13 @@
 
 def pytest_addoption(parser):
     group = parser.getgroup('slack')
+    group.addoption(
+        '--disable_ssl_verify',
+        action='store',
+        dest='disable_ssl_verify',
+        default=False,
```

Comment:
It's better tho have option and `--ssl_verify` - it's better to avoid negative options.
and set it by default = true